cumulus_library/base_utils.py

```diff
@@ -134,10 +134,19 @@ def update_query_if_schema_specified(
     query: str, manifest: study_manifest.StudyManifest
 ):
     if manifest and manifest.get_dedicated_schema():
-        query = query.replace(
-            f"{manifest.get_study_prefix()}__",
-            f"{manifest.get_dedicated_schema()}.",
-        )
+        # External queries in athena require a schema to be specified, so we
+        # just trim the prefix here.
+        # TODO: Move this to be a database defined function.
```

Comment:
require a schema to NOT be specified? Looks like you're not adding the schema in the external flow. Or does this comment mean that the schema is ALREADY specified elsewhere, and shouldn't be repeated here?

Reply:
the latter - lemme clarify